### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by albertony**
#### What is the purpose of this change?

Some small doc fixes here and there. Commit messages describes the individual changes.

#### Was the change discussed in an issue or in the forum before?

Partly related to #6267, where I accidentally merged a change in autogenerated section. There is one commit here that fixes that.

#### Checklist

- [X] I have read the [contribution guidelines](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#submitting-a-new-feature-or-bug-fix).
- [ ] I have added tests for all changes in this PR if appropriate.
- [ ] I have added documentation for the changes if appropriate.
- [X] All commit messages are in [house style](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#commit-messages).
- [x] I'm done, this Pull Request is ready for review :-)

### CODE REVIEW

File: docs/content/flags.md [hunk 1]
```diff
@@ -616,7 +616,7 @@ and may be set in the config file.
       --webdav-pass string                           Password (obscured)
       --webdav-url string                            URL of http host to connect to
       --webdav-user string                           User name
-      --webdav-vendor string                         Name of the Webdav site/service/software you are using
+      --webdav-vendor string                         Name of the WebDAV site/service/software you are using
```
Comment (ncw) [inline docs/content/flags.md:619]: This is an auto generated file I think

**Review by ncw — APPROVED** [review]
Very nice changes - thank you :-)

The cross referencing is a really nice touch.

Please merge when happy!

File: docs/content/flags.md (hunk 1 above)
Comment (albertony) [reply docs/content/flags.md:619]: Oh really, I didn't know that (or perhaps just completely forgot it). Thanks.

**Comment by albertony** [comment]
Thanks. I added some more similar fixes, but should be done now. Have looked over the changes several times, so I'm pretty sure it is correct, but I want to verify the live docs in browser also before merge.

**Comment by ncw** [comment]
> Thanks. I added some more similar fixes, but should be done now.

Great. LGTM :-)

> Have looked over the changes several times, so I'm pretty sure it is correct, but I want to verify the live docs in browser also before merge.

I'm sure you know how to do this, but just in case you don't you do with `make serve` - requires [hugo](https://gohugo.io/) to be installed and on the path.

**Pull request merged**